okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java

```diff
@@ -420,4 +429,21 @@ public void getEnabledModules(Tenant ten,
     }
     futures.all(mdl, fut);
   }
+
+  private void updateNewPermModuleTenants(String tenant, ModuleDescriptor md) {
+    InterfaceDescriptor id = md.getSystemInterface("_tenantPermissions");
+    if (id == null) {
+      return;
+    }
+    if (id.getVersion().equals("1.0")) {
+      newPermModuleTenants.remove(tenant);
+    } else {
+      newPermModuleTenants.put(tenant, Boolean.TRUE);
+    }
+  }
+
+  public ConcurrentMap<String, Boolean> getNewPermModuleTenants() {
```

Comment:
Can you replace "new" by a more descriptive word, for example "expanded"?

Reply:
Done.